src/mainwindow.cpp

```diff
@@ -2164,7 +2164,7 @@ void MainWindow::clearSearch()
     m_editorDateLabel->clear();
     m_textEdit->blockSignals(false);
 
-    m_proxyModel->setFilterFixedString(QStringLiteral(""));
+    m_proxyModel->setFilterFixedString(QString());
```

Comment:
what's the reason of this, is there any improvement ?

Reply:
No, though the empty string is generally recommended to be used with `QLatin1String` if necessary, otherwise `QString()`. The difference is that the latter will return a null string.